build.gradle

```diff
@@ -196,7 +196,7 @@ repositories {
   jcenter()
 }
 
-ext["rest-assured.version"] = '4.0.0'
+ext["rest-assured.version"] = '4.3.0'
```

Comment:
let's see how java8 will work. when tried with java11 it wasn't compatible even with suggested fix by library people